core/iwasm/aot/debug/elf.h

```diff
@@ -0,0 +1,368 @@
+/****************************************************************************
+ * include/elf.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@yamt, when introducing new files from other projects, we used to (1) add a file `LICENSE` under the related folder, (2) update the `<wamr-root>/ATTRIBUTIONS.md` to avoid the license issue. Could you do the similar operations? Thanks.

Reply:
done.
i included their LICENSE/NOTICE files as they are. if it's preferred to strip unrelated parts, please let me know.

Reply:
Yes, I found that LINCESE_NUTTX is really long, but we only need the first part, right? Can we only keep the content before L202?

Reply:
ok. i removed unrelated parts.